app/views/users/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>This page shows all users</h1>
```

Comment:
- [ ] Just as you added `app/views/users/index.html.erb` and `app/views/users/show.html.erb`.
Also, consider adding `app/views/posts/index.html.erb` and `app/views/posts/index.html.erb` files with relevant HTML inside as you added here.
requirement:
For each URL you should create:

A route.
An action in the correct controller.
A view file.